src/gantt.ts

```diff
@@ -1407,10 +1418,27 @@ module powerbi.extensibility.visual {
             return this.timeScale(task.end) - this.timeScale(task.start);
         }
 
-        private getTooltipForMilstoneLine(timestamp: number, milestoneTitle: string): VisualTooltipDataItem[] {
+        private getTooltipForMilstoneLine(
+            timestamp: number,
+            milestoneTitle: string): VisualTooltipDataItem[] {
+            let dateTime: string = new Date(timestamp).toLocaleDateString();
+
+            if (!milestoneTitle) {
+                switch (this.viewModel.settings.dateType.type) {
+                    case DateTypes.Second:
+                    case DateTypes.Minute:
+                    case DateTypes.Hour:
+                        milestoneTitle = "Now";
```

Comment:
Do the same for strings "Now" and "Today"

Reply:
DONE